spec/buffered-process-spec.coffee

```diff
@@ -117,6 +111,7 @@ describe "BufferedProcess", ->
       expect(stdout).toEqual ''
 
   it "calls the specified stdout callback only with whole lines", ->
+    # WINSPEC: Fails because command is too long, /bin/echo is *nix only, odd newlining and quoting
```

Comment:
A large text file that we `cat` on Linux/Mac and `type` on Windows would probably do it. I'll add that today.